src/main/java/Vibes.java

```diff
@@ -0,0 +1,42 @@
+import java.util.Scanner;
+
+public class Vibes {
+    public static void main(String[] args) {
+        String name = "Vibes";
+        String line = "\t---------------------------------------------------------------------------------------";
```

Comment:
Perhaps improve the layout of this line? Using wrapped lines for lines with more than 120 chars could help improve the readability of this code!